pages/getData.tsx

```diff
@@ -136,15 +212,31 @@ const GetData: NextPage = () => {
 
             <div className="field">
               <label className="label">Data key</label>
-              <div className="control">
+              <FormControl fullWidth>
```

Comment:
Let's get rid of Mui please, use standard bulma html:

https://bulma.io/documentation/form/general/

Please keep it as it was